release-please-config.json

```diff
@@ -32,7 +32,12 @@
           "type": "toml",
           "path": "Cargo.toml",
           "jsonpath": "$.workspace.dependencies.blackbox_solver.version"
-        }
+        },
+        {
+          "type": "json",
+          "path": "acvm_js/package.json",
+          "jsonpath": "$.version"
+        }        
       ]
```

Comment:
We should also have the Cargo.toml, no?

Reply:
Should get sorted as separate component in #424780c.